example/index.ts

```diff
@@ -4,15 +4,9 @@ import fs from 'fs/promises';
 import os from 'os';
 import path from 'path';
 import unzipper from 'unzipper';
-
-type IndexData = {
-    [key: string]: {
-        projects: string[];
-    };
-};
+import IndexData from '../test_data/projects/index.json';
```

Comment:
I would prefer to change the datatype than do the import of a .json file.

Reply:
I imagine an AI could easy discern the required change in datatype. :-)

Reply:
Is there a reason you don't want to infer the type from the file? We could do `import type IndexData from '../test_data/projects/index.json';` instead here and then read the file in from the file system at runtime like we were before.